lib/ts/recipe/authRecipe/components/feature/authPage/authPage.tsx

```diff
@@ -297,6 +301,7 @@ const AuthPageInner: React.FC<AuthPageProps> = (props) => {
                 abortCtl.signal
             ).then(() => {
                 lastBuild.current.buildReq = rebuildReqCount;
+                setOriginalFactorIds(authComponentListInfo?.factorIds);
```

Comment:
I may be wrong about this so please test it out but I think if `buildAndSetChildProps` calls `setAuthComponentListInfo` that will not be reflected here in `authComponentListInfo?.factorIds`